bootstrap.sh

```diff
@@ -14,6 +14,7 @@ sudo salt-call --local service.restart salt-minion
 sudo salt-key -A -y
 sudo salt -G 'roles:master' state.sls master_utils.libgit
 sudo salt -G 'roles:master' state.sls master_utils.bootstrap
+sudo salt -G 'roles:master' state.sls master_utils.nodegroups
```

Comment:
This line can be deleted now since that state got removed

Reply:
Aha, thanks for catching that.